test/lint/README.md

```diff
@@ -16,13 +16,15 @@ result is cached and it prevents issues when the image changes.
 test runner
 ===========
 
-To run all the lint checks in the test runner outside the docker, use:
+To run all the lint checks in the test runner outside the docker you first need
+to [install the rust toolchain](https://www.rust-lang.org/tools/install) locally.
```

Comment:
That was also my intention with this link. It recommends one installation method but makes clear this is just a recommendation and also links to [other installation methods available](https://forge.rust-lang.org/infra/other-installation-methods.html). What alternative solution would you suggest?

Also, I think generally in the docs we often just suggest one specific installation method for dependencies, like `brew` for macos or `apt-get` for Ubuntu etc. although there are other options available.

Reply:
>Not adding a link? Seems unlikely that someone doesn't know how to install packages, and at the same time can not use a search engine?

Anyone using a search engine will either end up on this page (first result) or another page that suggests the same install method, so we may save people that indirection and [keep them safer in the process too](https://en.wikipedia.org/wiki/Malvertising). 

How about just copying from the experts? https://github.com/rust-bitcoin/rust-bitcoin?tab=readme-ov-file#installing-rust, i.e. this: "Rust can be installed using your package manager of choice or [rustup.rs](https://rustup.rs/). The former way is considered more secure since it typically doesn't involve trust in the CA system. But you should be aware that the version of Rust shipped by your distribution might be out of date."

Reply:
Ok, then I am keeping this as is for now but I will remove the link if more reviewers share your opinion that it should be removed.

Reply:
done